src/model.py

```diff
@@ -59,7 +59,7 @@ def training_step(self, batch: TrainBatch, batch_index: int) -> Dict[str, Any]:
         mask = batch.mask.long()
         out = self(img)
         loss_val = self.loss_fn(out, mask)
-        self.log("train_loss", loss_val, on_step=True, on_epoch=True, prog_bar=True)
+        self.log("train_loss", loss_val, prog_bar=True, logger=True)
```

Comment:
the on_epoch and on_step isn't needed, so removed them to avoid confusion. The only thing that's needed is logger=True

Reply:
The only downside of this is that it doesn't show how to use self.logger.experiment, but there's [an example in the pytorch-lighting docs](https://pytorch-lightning.readthedocs.io/en/stable/logging.html#manual-logging)